apps/src/templates/feedback/CodeWritten.jsx

```diff
@@ -0,0 +1,46 @@
+var msg = require('../../locale');
+
+var CodeWritten = React.createClass({
+  
+  propTypes: {
+    numLinesWritten: React.PropTypes.number.isRequired,
+    totalNumLinesWritten: React.PropTypes.number.isRequired,
+  },
+
+  render: function () {
+    var lines, totalLines, showCode;
+    var styles = {
+      summary: {
+        fontSize: '18px',
+        lineHeight: '20px',
+        fontWeight: 'normal',
+        outline: 'none',
+        padding: '5px',
```

Comment:
Note: If you want, you can also just do `fontSize: 18` and React will automatically append 'px'.